README.md

```diff
@@ -30,13 +30,13 @@ transformers:
 * Run `java -jar deobfuscator.jar`
 * Re-run the detection if the JAR was not fully deobfuscated - it's possible to layer obfuscations
 
-Take a look at [USAGE.md](USAGE.md) for more information.
+Take a look at [USAGE.md](USAGE.md) or [wiki](./wiki) for more information.
 
 ## It didn't work
 
 If you're trying to recover the names of classes or methods, tough luck. That information is typically stripped out and there's no way to recover it.
 
-If you are using one of our transformers, check out the commonerrors folder to check for tips.
+If you are using one of our transformers, check out the [commonerrors](https://github.com/java-deobfuscator/deobfuscator/tree/master/commonerrors) folder to check for tips.
```

Comment:
I like this idea; however, this will link the master branch version of `commonerrors` regardless of whatever branch someone is looking at.